src/plugins/postcss-icss-parser.js

```diff
@@ -9,22 +9,22 @@ export default postcss.plugin(
   pluginName,
   () =>
     function process(css, result) {
-      const imports = {};
-      const icss = extractICSS(css);
-      const exports = icss.icssExports;
+      const importReplacements = Object.create(null);
+      const { icssImports, icssExports } = extractICSS(css);
 
-      Object.keys(icss.icssImports).forEach((key) => {
+      let index = 0;
+      Object.keys(icssImports).forEach((key) => {
         const url = loaderUtils.parseString(key);
 
-        Object.keys(icss.icssImports[key]).forEach((prop) => {
-          const index = Object.keys(imports).length;
+        Object.keys(icssImports[key]).forEach((prop) => {
+          index += 1;
```

Comment:
Why use `index`? Better solution will be good also

Reply:
I was refactoring the existing code which used the `index` the same way (I believe) but it was calculating it on every loop iteration. e.g. `Object.keys(imports).length`. This way does less work, and seemed like a harmless refactor

Reply:
:+1: